blazor/predefined-dialogs/getting-started.md

```diff
@@ -0,0 +1,330 @@
+
+> From 2022 Vol1 (20.1) version - The default value of `IgnoreScriptIsolation` is changed as `true`, so, you don’t have to set `IgnoreScriptIsolation` property explicitly to refer scripts externally.
+
+### Blazor Server App
+
+* For **.NET 6** app, open the **~/Program.cs** file and import Syncfusion Blazor Service and Sycnfusion Blazor Dialog Service.
+
+* For **.NET 5 and .NET 3.X** app, open the **~/Startup.cs** file and import Syncfusion Blazor Service and Sycnfusion Blazor Dialog Service.
+
+{% tabs %}
+{% highlight c# tabtitle=".NET 6 (~/Program.cs)" hl_lines="1 2 7 8" %}
+
+@using Syncfusion.Blazor
+@using Syncfusion.Blazor.Popups
+
+// Add services to the container.
+builder.Services.AddRazorPages();
+builder.Services.AddServerSideBlazor();
+builder.Services.AddScoped<SfDialogService>();
+builder.Services.AddSyncfusionBlazor(options => { options.IgnoreScriptIsolation = true; });
+
+var app = builder.Build();
+....
+
+{% endhighlight %}
+
+{% highlight c# tabtitle=".NET 5 and .NET 3.X (~/Startup.cs)" hl_lines="1 2 14 15" %}
+
+@using Syncfusion.Blazor
+@using Syncfusion.Blazor.Popups
+
+namespace BlazorApplication
+{
+    public class Startup
+    {
+        ...
+        public void ConfigureServices(IServiceCollection services)
+        {
+         	services.AddRazorPages();
+    		services.AddServerSideBlazor();
+    		services.AddScoped<SfDialogService>();
+            services.AddSyncfusionBlazor(options => { options.IgnoreScriptIsolation = true; });
+    		services.AddServerSideBlazor().AddCircuitOptions(options => { options.DetailedErrors = true; });
+    		services.AddServerSideBlazor().AddHubOptions(o =>
+        }
+        ...
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### Blazor WebAssembly App
+
+Open **~/Program.cs** file and register the Syncfusion Blazor Service and Sycnfusion Blazor Dialog Service in the client web app.
+
+{% tabs %}
+{% highlight C# tabtitle=".NET 6 (~/Program.cs)" hl_lines="1 2 7 8" %}
+
+@using Syncfusion.Blazor
+@using Syncfusion.Blazor.Popups
+
+// Add services to the container.
+
+builder.Services.AddScoped(sp => new HttpClient { BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) });
+builder.Services.AddScoped<SfDialogService>();
+builder.Services.AddSyncfusionBlazor(options => { options.IgnoreScriptIsolation = true; });
+await builder.Build().RunAsync();
+....
+
+{% endhighlight %}
+
+{% highlight c# tabtitle=".NET 5 and .NET 3.X (~/Program.cs)" hl_lines="1 2 11 12" %}
+
+@using Syncfusion.Blazor
+@using Syncfusion.Blazor.Popups
+
+namespace WebApplication1
+{
+    public class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            ....
+            builder.Services.AddScoped<SfDialogService>();
```

Comment:
need to highlight this line or add details in the comment line

Reply:
Hi @saravanangn 

I have highlighted the line in 118 line. 

<img width="529" alt="image" src="https://user-images.githubusercontent.com/93525217/182094015-b72dd58a-0b2d-42a4-b98d-afc9398784eb.png">